packages/calling/src/Events/types.ts

```diff
@@ -372,3 +374,16 @@ export type SanitizedEndTimeAndSessionId = {
   endTime: number;
   sessionId: string;
 };
+export type Line = {
+  dnorpattern: string;
+  index: number;
+  label: string | null;
+};
+export type Device = {
+  name: string;
+  model: number;
+  lines: Line[];
+};
+export type LinesApiResponse = {
```

Comment:
Can we change all these types to say UCM. 
Line -> UCMCLine or UCMLine
Since we already have line and device types. This will allow us to differentiate.

Reply:
Done